packages/inventories-exporter/test/main.spec.js

```diff
@@ -0,0 +1,359 @@
+import streamtest from 'streamtest'
+import { stripIndent } from 'common-tags'
+import InventoryExporter from '../src/main'
+
+describe('InventoryExporter', () => {
+  it('should be defined', () => {
```

Comment:
that is pretty peculiar. If I have to test that my function is defined, then my export was pretty bad in the first place..
but okay, there is no harm to test that it is defined